src/LibEventLoop.php

```diff
@@ -298,7 +298,7 @@ private function unsubscribeStreamEvent($stream, $flag)
      */
     private function createTimerCallback()
     {
-        $this->timerCallback = function ($_, $_, $timer) {
+        $this->timerCallback = function ($_, $__, $timer) {
```

Comment:
Yep, right. The code crashes because of this line when you try to include
this class. That's something that static analysis tools can stumble upon.

So this fixes that. But there's not actually a way to run this
implementation on PHP 7.

On Tuesday, 1 March 2016, Christian Lück <notifications@github.com> wrote:

> In src/LibEventLoop.php
> <https://github.com/reactphp/event-loop/pull/45#discussion_r54551382>:
>
> > @@ -298,7 +298,7 @@ private function unsubscribeStreamEvent($stream, $flag)
> >       */
> >      private function createTimerCallback()
> >      {
> > -        $this->timerCallback = function ($_, $_, $timer) {
> > +        $this->timerCallback = function ($_, $__, $timer) {
>
> I see why we would want this change, but for the reference only: Afaict
> this line is currently untested because the libevent extension does not
> compile under PHP 7, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reactphp/event-loop/pull/45/files#r54551382>.
>


-- 

Ondřej Mirtes